doc/advanced/understanding_memory_formats.md

```diff
@@ -115,9 +115,9 @@ in this example.
 
 One can create memory with **NCHW** data layout using
 #dnnl_nchw of the enum type #dnnl_format_tag_t defined in
-[dnnl_types.h](https://github.com/oneapi-src/oneDNN/blob/master/include/oneapi/dnnl/dnnl_types.h)
+[dnnl_types.h](https://github.com/uxlfoundation/oneDNN/blob/master/include/oneapi/dnnl/dnnl_types.h)
 for the C API, and dnnl::memory::format_tag::nchw defined in
-[dnnl.hpp](https://github.com/oneapi-src/oneDNN/blob/master/include/oneapi/dnnl/dnnl.hpp)
+[dnnl.hpp](https://github.com/uxlfoundation/oneDNN/blob/master/include/oneapi/dnnl/dnnl.hpp)
```

Comment:
nit: should we change the branch name from `master` to `main`? This link still works and re-directs to `main` branch automatically.

Reply:
Good catch. We definitely should.